src/styles/block/clock.scss

```diff
@@ -0,0 +1,54 @@
+@mixin handSize {
+  position: absolute;
+  bottom: 50%;
+  left: 40%;
+  border-radius: 10px;
+  width: 5px;
+}
```

Comment:
you should create _mixin.scss file for this code

Reply:
Agree, let's move this block to `_mixin.scss` file